# represents all YOLOv8 detections in one frame

std_msgs/Header header
yolov8/Detection[] detections